project/plugins.sbt

```diff
@@ -1,8 +1,10 @@
-addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.1")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "latest.integration")
 
-addSbtPlugin("org.scala-lang.modules.scalajs" %% "scalajs-sbt-plugin" % "0.4.3")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "latest.integration")
```

Comment:
Using `latest.integration` is considered bad practice in open source projects, as it can break the build at any moment. It's even worse for code that depends on this library, as now they *also* get their build broken at random points in time.